### DISCUSSION TITLE
seeking some guidance and advice on addressing accessibility challenges with my website.

### DISCUSSION THREAD

**Discussion by JISSO90** [community/community#126887] [Programming Help]
I've realized that my website lacks alt text for images, essential for users relying on screen readers. I understand the importance of providing descriptive alt text to make the content accessible to all users, and I'm looking for guidance on implementing this effectively.
My website URL is https://gameavatr.com/avatar-world-mod-apk-for-ios/

**Comment by goldieowner17** [comment]
Hey there! 👋

Thanks for posting in the GitHub Community, @JISSO90  ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. 

I've gone ahead and moved this to the correct category for you. Good luck!

**Comment by CopperEagle** [comment]
Hey @JISSO90 

You may be interested in the w3 guidelines for accessible webpages, WCAG2. [Here](https://www.w3.org/WAI/standards-guidelines/wcag/glance/) is the overview. There are also extensions for Firefox and Chrome that test a website for accessibility issues. A list (a few years old but still fine) is [here](https://medium.com/@OPTASY.com/what-are-some-of-the-best-web-accessibility-testing-tools-to-evaluate-your-website-with-69def25a386).

Instead of positing links to some unknown webpages, you could provide links to repos. At least as far as I am concerned, I tend to be more cautios where I click since there is so much spam in the community discussions right now. No offense, ofc.

**Reply by ebndev** [reply]
@JISSO90 this is great advice above! Let us know if you were able to make any accessibility changes to your site 😄
